### PR TITLE
Translation util

### LINKED ISSUE #176: Add translation util

### PR DESCRIPTION
**Pull request by jswale**
fixes #176

### CODE REVIEW

File: src/utils/translations.ts [hunk 1]
```diff
@@ -0,0 +1,85 @@
+import { getAssetPath } from "@stencil/core";
+import { appVersion } from "~global/init";
```
Comment (vincentv) [inline src/utils/translations.ts:2]: "~/global/init.ts" is there to configure the environment. It is not supposed to be imported into a file.

Comment (jswale) [reply src/utils/translations.ts]: Done

File: src/global/init.ts [hunk 2]
```diff
@@ -6,6 +6,8 @@ import {
   faHome,
   faQuestionCircle,
 } from "@fortawesome/free-solid-svg-icons";
+import * as CacheUtils from "~utils/cache";
```
Comment (vincentv) [inline src/global/init.ts:9]: you don't need to import the world, you only use setAppVersion

```typescript
import { setAppVersion } from "~utils/cache";
```

File: src/utils/cache.spec.ts [hunk 3]
```diff
@@ -0,0 +1,25 @@
+import * as CacheUtils from "~utils/cache";
```
Comment (vincentv) [inline src/utils/cache.spec.ts:1]:
```typescript
import { getITem, removeITem, setAppVersion, setItem} from "~utils/cache"
```

File: src/utils/translations.ts [hunk 4]
```diff
@@ -0,0 +1,84 @@
+import { getAssetPath } from "@stencil/core";
+import * as CacheUtils from "~utils/cache";
```
Comment (vincentv) [inline src/utils/translations.ts:2]: avoids this kind of import
fyi : [tree shaking like a pro](https://medium.com/@bluepnume/javascript-tree-shaking-like-a-pro-7bf96e139eb7#43e1)

File: src/global/init.ts (hunk 2 above)
Comment (jswale) [reply src/global/init.ts:9]: done

File: src/utils/cache.spec.ts (hunk 3 above)
Comment (jswale) [reply src/utils/cache.spec.ts]: done

File: src/utils/translations.ts (hunk 4 above)
Comment (jswale) [reply src/utils/translations.ts]: done

**Pull request merged**